Source/WebKit/Platform/cocoa/LayerHostingContext.mm

```diff
@@ -112,20 +127,40 @@
 
 LayerHostingContext::~LayerHostingContext()
 {
+#if USE(EXTENSIONKIT)
+    [m_hostingUpdateCoordinator commit];
+#endif
 }
 
 void LayerHostingContext::setRootLayer(CALayer *rootLayer)
 {
+#if USE(EXTENSIONKIT)
+    if (m_hostable) {
+        m_hostable.get().layer = rootLayer;
```

Comment:
Nit: this is fine, but our style preference in these cases is to omit the `.get()` by writing `[m_hostable setLayer:rootLayer]`.